project/Dependencies.scala

```diff
@@ -23,29 +23,29 @@ object Dependencies {
 
   val scalaJava8Compat = Seq("org.scala-lang.modules" %% "scala-java8-compat" % "0.8.0")
 
-  val playJsonVersion = "2.7.2"
+  val playJsonVersion = "2.7.4"
   val playJson = Seq("com.typesafe.play" %% "play-json" % playJsonVersion)
 
   val slf4jApi = Seq("org.slf4j" % "slf4j-api" % "1.7.26")
 
   val javaxInject = Seq("javax.inject" % "javax.inject" % "1")
 
-  val sslConfigVersion = "0.3.7"
+  val sslConfigVersion = "0.4.0"
```

Comment:
I'm not sure 0.4.0 and 0.3.7 are compatible. Maybe we will need to cut a release from 0.3.7 that supports Scala 2.13.

@dwijnand what do you think?

Reply:
It's possible they are, but it's safer to try to crossbuild 0.3.7.